kubernetes/stage/deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: moderator-web
-        image: 783633885093.dkr.ecr.us-west-2.amazonaws.com/moderator:bbae8c1be8431ace923c58caac928917fc2aa31e
+        image: 783633885093.dkr.ecr.us-west-2.amazonaws.com/moderator:6b7f9e15a1f4167cfdab2ea426fb7c8b2806e899
```

Comment:
hiya @akatsoulas - i don't see an image in moderator ECR with this tag. What image / build are you referring to?

Reply:
I see the commit and the docker image relevant to it - i had changed in the GHA migration to tag docker images with the shortened SHA - that's my bad I didn't make it clearer or introduced a change at all. Manually retagging the correct image with the full SHA now, then looking into required db migration for ya.